Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/HeadersValidationTests.cs

```diff
@@ -306,6 +306,191 @@ private void ValidateJsonSerializationFormatQuery(DocumentClient client, Documen
             Assert.IsTrue(response.ResponseBody.ReadByte() == HeadersValidationTests.BinarySerializationByteMarkValue);
         }
 
+        [TestMethod]
+        public void ValidateSupportedSerializationFormatsGateway()
+        {
+            DocumentClient client = TestCommon.CreateClient(true);
```

Comment:
```suggestion
            using DocumentClient client = TestCommon.CreateClient(true);
```

Dispose clients to avoid high resource consumption of running clients

Reply:
Leaking clients leave background account refreshes running in the back

Reply:
updated.